### ISSUE TITLE
Error: ENOENT: no such file or directory.

### ISSUE THREAD

**Issue by xyifer12** [SteamAchievementNotifier/SteamAchievementNotifier#19]
I'm back with a cool new issue!
Issue: Multiple error messages appeared when a specific achievement registered as unlocked.

Information:
The error seems to be related to the fact that a question mark is in the name. SAN has failed to save a screenshot for this specific achievement. I run Windows 10 Pro 21H2, x64, SAN 1.84. GeForce Experience is always passively recording.
![error](https://user-images.githubusercontent.com/33806276/173165409-941890f4-4903-47db-b13f-eedfe74a9ca8.png)

**Comment by SteamAchievementNotifier** [comment]
Hey again @xyifer12,

That sounds about right, I'll test this out and add the fix in the next App Revision.

Thanks for letting me know!

**Comment by SteamAchievementNotifier** [comment]
Hey @xyifer12,

I've addressed this issue in App Revision 0.6 that was released yesterday (this should be automatically downloaded when starting the application via the SANLauncher EXE). I'll close this issue as I think you probably won't encounter it too often, but all unsupported characters in filenames have now been excluded from the file path generated by the "Save Screenshots with Overlay" feature.

Thanks!
